bigtable-hbase-parent/bigtable-hbase-mapreduce/pom.xml

```diff
@@ -33,54 +33,125 @@ limitations under the License.
     </description>
 
     <properties>
-        <hbase.version>1.1.2</hbase.version>
+        <hbase.version>1.2.4</hbase.version>
+        <hadoop.version>2.7.3</hadoop.version>
     </properties>
 
     <dependencies>
         <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <scope>compile</scope>
         </dependency>
+
         <dependency>
             <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-common</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hbase</groupId>
-            <artifactId>hbase-annotations</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hbase</groupId>
-            <artifactId>hbase-common</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hbase</groupId>
-            <artifactId>hbase-client</artifactId>
+            <artifactId>hadoop-client</artifactId>
+            <version>${hadoop.version}</version>
+            <scope>provided</scope>
         </dependency>
+
         <dependency>
-            <groupId>com.google.cloud.bigtable</groupId>
-            <artifactId>bigtable-hbase-1.1</artifactId>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>bigtable-hbase-1.2</artifactId>
             <version>${project.version}</version>
+
             <exclusions>
                 <exclusion>
-                    <groupId>${project.groupId}</groupId>
-                    <artifactId>bigtable-hbase-shaded</artifactId>
+                    <groupId>io.netty</groupId>
```

Comment:
Are these required outside of Dataproc, if so, then the exclusion list should probably be a set ofshade plugin artifactSet/exclusions